doc/rtd/explanation/about-cloud-config.rst

```diff
@@ -0,0 +1,85 @@
+.. _about-cloud-config:
+
+About the cloud-config file
```

Comment:
Q: Is the title helpful/descriptive enough?

Reply:
Looks good enough as this is a general topic that covers how #cloud-config files are interpreted.